database/src/main/postgres/flows/V1.7.5__get_flow_checkpoints.sql

```diff
@@ -0,0 +1,127 @@
+/*
```

Comment:
Make the file name `V1.8.3_...` or `V1.9.1_...`
And dependent on #167

Reply:
damned, this really hurts :D

Reply:
I chose 1.9.1 so that it's less schysophrenic